src/main/haskell/kore/app/prover/Main.hs

```diff
@@ -32,11 +47,41 @@ pCommand' = parseCommand parseName parseFormula parseRule
                                parseName
 
 proveCommand
-    :: IO (ProverState
-            String
-            (MLRule (SymbolOrAlias Meta) (Variable Meta) CommonMetaPattern)
-            CommonMetaPattern)
-proveCommand = runProver dummyFormulaVerifier pCommand' (ProverState emptyProof)
+  :: (Reifies sig ValidatedSignature)
```

Comment:
Where is this `Reifies` constraint used?

Reply:
This is some leftover stuff from brandon's implementation that I didn't understand so I left it for now